extensions-contrib/graphite-emitter/src/main/java/io/druid/emitter/graphite/GraphiteEmitterConfig.java

```diff
@@ -50,6 +50,9 @@
   final private List<String> alertEmitters;
 
   @JsonProperty
+  final private List<String> requestLogEmitters;
```

Comment:
Is there a way to set this to `null`? it seems there are a few spots in the code that will break if this ends up being `null`